redisinsight/ui/src/pages/home/components/manual-connection/manual-connection-form/ManualConnectionForm.tsx

```diff
@@ -229,47 +213,40 @@ const ManualConnectionForm = (props: Props) => {
     )
   }
 
+  const Tabs = () => (
+    <EuiTabs className={cx('tabs-active-borders', styles.tabs)}>
+      {MANUAL_FORM_TABS.map(({ id, title }) => (
```

Comment:
Note: we should configure the linter to cry about `key`. No action needed here.

Reply:
good point, fixed